rest-api/tools/auth_setup.sh

```diff
@@ -94,7 +91,7 @@ function get_db_password {
 function get_geocode_key {
   echo "Getting geocode api key ..."
   tools/install_config.sh --key geocode_key --instance $INSTANCE \
-       --config_output "$TMP_GEOCODE_INFO_FILE"
+       --creds_file ${CREDS_FILE} --project ${PROJECT} --config_output "$TMP_GEOCODE_INFO_FILE"
```

Comment:
As mentioned over slack -- I think we should just hardcode --project to pmi-drc-api-test here. That way we can use the same geocode key for all cloud environments, as well as for local development.